doc/NextGen_ON_CONUS.md

```diff
@@ -0,0 +1,174 @@
+# NextGen on CONUS
+
+This documentation provides instructions on all neccessary steps and components to run NextGen jobs at CONUS scale. Considering the computation's large scale, we focus only on running parallel jobs using MPI.
+
+* [Summary](#summary)
+* [Download the Codes](#doenload-the-codes)
+* [Setting Up the Environment](#setting-up-the-environment)
+* [Build the Executable](#build-the-executable)
+* [CONUS Hydrofabric](#CONUS-hydrofabric)
```

Comment:
This doesn't link in the rich diff, does changing the URL slightly to `#conus-hydrofabric` resolve that?

```suggestion
* [CONUS Hydrofabric](#conus-hydrofabric)
```

Reply:
Surprisingly, this one does seem to work.

Reply:
The correction makes the URL more consistent with other sections.